charts/enterprise-logs/CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ Entries should be ordered as follows:
 
 Entries should include a reference to the pull request that introduced the change.
 
+## Unreleased
+
+## 1.2.1
+
+- [BUGFIX] Fixed the MinIO endpoints in the resultiung `config.yaml`, which pointed incorrectly to `{{ .Release.Name }}-enterprise-logs-minio` instead of `{{ .Release.Name }}-minio`. #826
```

Comment:
There's a typo in "resulting". 

nit: I would consider something simpler like:
```suggestion
- [BUGFIX] Fixed the dev cluster MinIO endpoints in default configuration. #826
```
or just replacing the reference to `config.yaml` with something like "GEL configuration file" since the latter is more likely to be understood by users.